### ISSUE TITLE
Rain Sensor - Entity "Last hour rain" duplicate

### ISSUE THREAD

**Issue by janoschbatschi** [PlusPlus-ua/ha_mobilealerts#14]
First of all, thanks for your work! I was looking for that integration for a long time. 

There seems to be an issue with the rain sensor. There are two entries for "last hour rain" but there is no one for "last day rain". I didn't have the chance to check the values - could be a naming issue and the second "last hour rain" has the data for "last day rain". 

<img width="328" alt="image" src="https://github.com/PlusPlus-ua/ha_mobilealerts/assets/2529951/09be7776-c3a8-44e3-b9a0-b216838e79d2">

**Comment by janoschbatschi** [comment]
Obviously, there is not only a naming issue, but there is also no data on last hour rain or last day rain. Today we had rain twice and there was no data reported (the app reported the data and also the accumulated rain data was updated - compare screenshot)
<img width="337" alt="image" src="https://github.com/PlusPlus-ua/ha_mobilealerts/assets/2529951/ceb635e0-d13a-43e2-9831-522e21c02de7">

**Comment by hiatsu0** [comment]
I have wondered this as well, and I recall reading somewhere that there was some naming issue, hopefully fixed in an upcoming version. 

But indeed the functionality of these entries _might_ have some issues as well: I recall seeing just once some data in the "Last rain" entity, and even that disappeared shortly after the actual rain (maybe an hour or so). I think something like "Last rain" should have a little longer timespan..maybe like a few days (or even no time limit at all, and the date & time of the last detected rainfall as an attribute, in addition to the millimeters)
